PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,30 @@
-Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).
+<!-- Thanks contribution a pull request! Here are some tips for you:
 
-When filling in a section please remove the help text and the above text.
+1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
+2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
+3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
+-->
 
 #### Summary
-[A brief description of what this pull request does.]
+<!--
+A description of what this pull request does.
+-->
 
 #### Ticket Link
-[Please link the GitHub issue or Jira ticket this PR addresses.]
+<!--
+Please link the GitHub issue, if this PR addresses a Help Wanted ticket e.g.
+
+Fixes https://github.com/mattermost/mattermost-server/issues/XXXXX
+
+or otherwise the Jira ticket.
+-->
 
 #### Checklist
-[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
-- [ ] Added or updated unit tests (required for all new features)
-- [ ] Added API documentation (required for all new APIs)
-- [ ] All new/modified APIs include changes to the drivers
-- [ ] Has enterprise changes (please link)
-- [ ] Has UI changes
-- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-server/blob/master/i18n/en.json)) updates
-- [ ] Touches critical sections of the codebase (auth, upgrade, etc.)
+<!--
+Please check every fields. Please remove unrelated fields.
+-->
+- Run test on the related changes locally
```

Comment:
Do we have instructions on how to run these tests? E.g. running `make test`

Reply:
There isn't a simple one liner for this. But we can link to https://developers.mattermost.com/contribute/server/developer-workflow/.

```suggestion
- Run test on the related changes locally <!-- See https://developers.mattermost.com/contribute/server/developer-workflow/ for an explanation how to do this -->
```

Reply:
The problem is that `make test` runs _all_ test which takes some time and can even be to much for a weaker computer. Running just individual tests is usually recommended.

Reply:
Ah okay. In that case we could link to https://developers.mattermost.com/contribute/server/developer-workflow/#running-only-specific-server-unit-tests

```
Tested related changes <!-- See https://developers.mattermost.com/contribute/server/developer-workflow/#running-only-specific-server-unit-tests how to run specific unit tests. -->
```